app/pages/lab/visibility.cjsx

```diff
@@ -169,6 +169,9 @@ module.exports = React.createClass
             {@state.workflows.map (workflow) =>
               setting = workflow.active
               stats_completeness_type = workflow.configuration.stats_completeness_type ? 'retirement'
+              stats_visible = !workflow.configuration.stats_hidden
+              if !workflow.active
+                stats_visible = !(workflow.configuration.stats_hidden == undefined) && stats_visible
```

Comment:
This is the confusing bit that is needed so *inactive* workflows are hidden by default (i.e. `stats_hidden` is not set, but *active* workflows should be shown by default.